shared/domains/chelonia/internals.js

```diff
@@ -1298,7 +1307,7 @@ export default (sbp('sbp/selectors/register', {
       config[`postOp_${opT}`]?.(message, state) // hack to fix syntax highlighting `
     }
   },
-  'chelonia/private/in/enqueueHandleEvent': function (contractID: string, event: string) {
+  'chelonia/private/in/enqueueHandleEvent': (contractID: string, event: string) => {
```

Comment:
Why is this change being made? These are supposed to be un-bound functions so that they have access to the chelonia namespace via `this`